README.md

```diff
@@ -185,7 +185,7 @@ You can also see [an example of using `kubeconfig` in the `./extras`](./extras/e
 
 Once you have it running, you can hit any of the nodes (port 30080 in the example above) and see the UI.
 
-![](./extras/screenshot.png)
+![](./extras/screenshot-big.png)
```

Comment:
Why not have both side by side?

Reply:
The other one is already at the top of the page ?

Reply:
...yes. Ignore me.